src/main/java/app/RunGestureBridgeApp.java

```diff
@@ -104,9 +103,8 @@ private static VoiceSettingsView initializeSettings() {
      * @return the TextToSpeechController instance.
      */
     private static TextToSpeechController initializeTextToSpeech(ViewInterface gestureBridgeView) {
-        AudioPlayer audioPlayer = new AudioPlayer();
         TextToSpeechInterface textToSpeechService = new GoogleTextToSpeechGateway();
-        TextToSpeechOutputBoundary outputBoundary = new TextToSpeechPresenter(audioPlayer);
+        TextToSpeechOutputBoundary outputBoundary = new TextToSpeechPresenter(gestureBridgeView);
```

Comment:
I was initially a bit confused why the presenter does not have a dependency relation with the view. Now that it's modified with the audio played within the view, the structure makes a lot more sense and follows CA better!